### DISCUSSION TITLE
Use sedutil instead of luks

### DISCUSSION THREAD

**Discussion by sluramod** [noahbliss/mortar#38] [Ideas]
Hello,

Is it possible to use a similar structure to support [sedutil](https://github.com/Drive-Trust-Alliance/sedutil) of LUKS?

For example, leave ESP partition read-only and unecrypted, and store password required to decrypt locking range for the main partition in TPM? If so, what additions would be needed in mortar?

Regards,
Alex

**Comment by noahbliss** [comment]
Hey! Sorry for the delay. This may be possible but I lack the equipment for this use case currently. You're welcome to fork and give it a shot though!
